docs/overlays/elasticsearch-shared-overlays.yaml

```diff
@@ -4,6 +4,274 @@ info:
   title: Overlays for changes that apply to both Elasticsearch and Elasticsearch Serverless OpenAPI documents
   version: 0.0.1
 actions:
+  # Add an alphabetically sorted list of tags
+  - target: '$'
+    description: Add document-level tags sorted by display name
+    update:
+      tags:
+      # A
+        - name: autoscaling
+          x-displayName: Autoscaling
+      # C 
+        - name: cat
+          x-displayName: Compact and aligned text (CAT)
+          description: >
+            The compact and aligned text (CAT) APIs aim are intended only for human consumption using the Kibana console or command line.
+            They are not intended for use by applications.
+            For application consumption, it's recommend to use a corresponding JSON API.
+            
+            All the cat commands accept a query string parameter `help` to see all the headers and info they provide, and the `/_cat` command alone lists all the available commands.
+        - name: cluster
```

Comment:
unsure why, but items at the `/_component_template` endpoint are displaying as ` cluster` items. I see that reflected [in the tag fix PR](https://github.com/elastic/elasticsearch-specification/pull/2889/files#diff-3a3f55821e7fc4208762783b2a0635ea25a647938275ac13931747a2e8b6498dR3227), but this one doesn't make as much logical sense to me. Could the tags be updated in cases like this?

Reply:
That seems to be derived from the original spec in https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/api/cluster.get_component_template.json, though I'm not sure why. It's another scenario that we could override if https://github.com/elastic/elasticsearch-specification/pull/2961 works

Reply:
I've added @doc_tag for those component template APIs in 3cba8c9210e49d97a8566737fbdeccd6634dd878